conformance/service_import.go

```diff
@@ -231,8 +231,11 @@ var _ = Describe("", func() {
 					})
 					Expect(serviceImport).NotTo(BeNil(), "ServiceImport was not found")
 
-					Expect(serviceImport.Annotations).To(Equal(helloServiceExport.Spec.ExportedAnnotations), reportNonConformant(""))
-					Expect(serviceImport.Labels).To(Equal(helloServiceExport.Spec.ExportedLabels), reportNonConformant(""))
+					Expect(serviceImport.Annotations).To(HaveKeyWithValue("dummy-annotation", "true"), reportNonConformant(""))
+					Expect(serviceImport.Annotations).To(Not(ContainElements("dummy-annotation2")), reportNonConformant(""))
+
```

Comment:
I think it would be cleaner to generalize these assertions:
                 
```
 assertHasKeyValues := func(actual, expected map[string]string) {
    for k, v := range expected {
        Expect(actual).To(HaveKeyWithValue(k, v), reportNonConformant(""))
    }
}

assertNotHasKeyValues := func(actual, expected map[string]string) {
    for k, v := range expected {
        Expect(actual).ToNot(HaveKeyWithValue(k, v), reportNonConformant(""))
    }
}
...
			
assertHasKeyValues(serviceImport.Annotations, helloServiceExport.Spec.ExportedAnnotations)

assertNotHasKeyValues(serviceImport.Annotations, helloServiceExport2.Spec.ExportedAnnotations)
```
		
Same elsewhere.

Reply:
Thanks! indeed makes sense, I just did this change let me know if there's anything else!